api-gateway-routes/api_gateway_proxy_function.rb

```diff
@@ -41,10 +41,10 @@ def on_connect(event, context)
     :queueUrl => sqs_queue.queue_url,
     :apiEndpoint => api_endpoint,
     :connectionId => request_context["connectionId"],
-    :projectUrl => authorizer["project_url"],
     :levelId => authorizer["level_id"],
     :options => authorizer["options"],
-    :iss => authorizer["iss"]
+    :iss => authorizer["iss"],
```

Comment:
What does iss stand for?

Reply:
issuer, it's a 'reserved claim' for a jwt token so we can't make it more specific

Reply:
Got it. If we're generally using this variable as the dashboard hostname, it might be nice to rename it for readability.

Reply:
If I'm understanding correctly, we can't rename the value from the authorizer, but can rename the value we pass in the payload? If it's straightforward to do that rename as part of this change, I agree that it would be nice. I keep having to remind myself what iss means too.